tests/test_commands.py

```diff
@@ -305,6 +305,21 @@ def test_asyncio_enabled_false(self):
         log = self.get_log(self.debug_log_spider, args=[])
         self.assertNotIn("Using reactor: twisted.internet.asyncioreactor.AsyncioSelectorReactor", log)
 
+    def test_custom_asyncio_loop_enabled_true(self):
+        log = self.get_log(self.debug_log_spider, args=[
+            '-s', 'TWISTED_REACTOR=twisted.internet.asyncioreactor.AsyncioSelectorReactor', '-s', 'ASYNCIO_EVENT_LOOP=uvloop.Loop'
+        ])
+        self.assertIn("Using asyncio event loop: uvloop.Loop", log)
+
+    def test_custom_asyncio_loop_enabled_false(self):
+        log = self.get_log(self.debug_log_spider, args=[
+            '-s', 'TWISTED_REACTOR=twisted.internet.asyncioreactor.AsyncioSelectorReactor'
+        ])
+        if sys.platform == 'win32':
+            self.assertIn("Using asyncio event loop: asyncio.windows_events._WindowsSelectorEventLoop", log)
+        else:
+            self.assertIn("Using asyncio event loop: asyncio.unix_events._UnixSelectorEventLoop", log)
```

Comment:
This may still be a problem, e.g. given https://github.com/tornadoweb/tornado/issues/2751

What I had in mind was using something like `new_event_loop()`, assuming that it returns an instance of the default loop class regardless of the loop installed in the Twisted reactor, and making sure that the reported asyncio event loop is of the same class.

Reply:
I have used `new_event_loop()` to find the default asyncio loop according to the platform. I think it  should work now.